definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -1,5 +1,6 @@
 [
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateLegalRepClaim", "UserRole": "citizen", "CRUD": "CR"},
```

Comment:
should we create a new role for legal rep?

Reply:
I think legal rep users are using `caseworker-cmc` role